### ISSUE TITLE
Support monorepo with NPM Workspaces

### ISSUE THREAD

**Issue by marlun78** [fastly/cli#598]
**Is your feature request related to a problem? Please describe.**

Currently, it is not possible to run `fastly compute serve` inside a monorepo with NPM Workspaces as demonstrated by [this sample repo](https://github.com/marlun78/fastly-error). A `@fastly/js-compute not found in package.json` error is thrown. 

```
➜  fastly-error git:(main) npm run start:client   

> fastly-error@1.0.0 start:client
> npm start -w packages/client

> @package/client@0.3.0 start
> fastly compute serve

✓ Initializing...
✓ Verifying package manifest...
✗ Verifying local javascript toolchain...

ERROR: `@fastly/js-compute` not found in package.json.

To fix this error, run the following command:

        $ npm install --save-dev @fastly/js-compute
npm ERR! Lifecycle script `start` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: @package/client@0.3.0 
npm ERR!   at location: [...]/fastly-error/packages/client 
```

I guess the issue is that when in a monorepo, the `node_modules` is on the root level and the relevant script runs in the context of a child-package (where also the relevant `package.json` is).

**Describe the solution you'd like**

I don't know the reason it's build as it is today, but I've never before come across another NPM package that inspects the package.json file to make sure dependencies are added. It seems to me to miss the target. The relevant question is rather if the dependency is _installed_ or not and the best way to find out is by using it, right?

**Additional context**

[NPM Workspaces](https://docs.npmjs.com/cli/v8/using-npm/workspaces)
Related to package [@fastly/js-compute](https://www.npmjs.com/package/@fastly/js-compute)
Related to issue #359

**Comment by Integralist** [comment]
Hi @marlun78 

We've been working on the following PR for a while now to try and address issues with the Fastly CLI build process:
https://github.com/fastly/cli/pull/640 

Once this is merged (and released) I'll ping back in this ticket so you can upgrade and see if the changes work better for your situation. If not, then we can continue the discussion.

Thanks.

**Comment by Integralist** [comment]
Hi @marlun78 

Apologies, I just realised I said I would report back once #640 had been merged. Since that PR was merged I was discussing some issues internally with a colleague and so https://github.com/fastly/cli/pull/671 is currently in the works which I think might be actually resolve this issue.

Just an FYI that this issue will be closed automatically once that PR is merged.

**Comment by Integralist** [comment]
Closing due to https://github.com/fastly/cli/pull/765 (this issue would have been automatically closed but I had to create a new PR and I neglected to include a reference to this issue).

**Comment by domdomegg** [comment]
@Integralist could you clarify how to use #765?

Unfortunately hitting the same issue, also in an NPM workspace.

**Comment by Integralist** [comment]
Hi @domdomegg 

I'm sorry to hear you are having issues.

Since this issue was closed the CLI has undergone some fundamental changes that have resulted in us stripping out a bunch of 'user environment' validation logic (see [this PR](https://github.com/fastly/cli/pull/785) recently merged). I haven't got these changes out into a new release yet (they'll be part of a 6.0.0 release of the CLI) but I believe that PR will help provide much greater flexibility to users in general.

**Comment by domdomegg** [comment]
Ah, thanks for the update! I'll wait for 6.0.0 in that case.

For now I've just disabled the verification for now, with:

```
fastly compute serve --skip-verification
```
